components/homepage-search/element.css

```diff
@@ -1,36 +1,33 @@
 .mdn-homepage-search {
   display: flex;
 
-  gap: 0.5ch;
-  justify-content: center;
+  gap: 0.25em;
+  align-items: center;
 
-  padding: 1em;
+  padding: 0.8em 1em;
   margin: 0 auto;
 
   font-size: var(--font-size-large);
 
-  border-color: var(--color-border-primary);
-  border-style: solid;
-  border-radius: 2em;
+  cursor: pointer;
 
-  &::before {
-    display: inline-block;
+  background-color: transparent;
+  border: 2px solid var(--color-border-primary);
+  border-radius: calc(infinity * 1px);
```

Comment:
`calc(infinity * 1px)` isn't baseline green (Firefox 114 released 2023-06-06), and looks like a weird hack anyway - I had to pause for a moment to figure out what was going on.

Reply:
It’s not the only place we use infinity to achieve a reliable pill-shaped button, no matter how big it is (hence the infinity), so maybe we should file an issue and merge this one as is?

I would introduce something like `--radius-pill: 9999px` for cases like this, since infinity is not green enough yet.